go/vt/vttls/vttls.go

```diff
@@ -203,3 +209,62 @@ func doLoadTLSCertificate(cert, key string) error {
 
 	return nil
 }
+
+var combinedTlsCertificates = sync.Map{}
+
+func combineAndLoadTLSCertificates(ca, cert, key string) (*[]tls.Certificate, error) {
+	combinedTlsIdentifier := tlsCertificatesIdentifier(ca, cert, key)
+	once, _ := onceByKeys.LoadOrStore(combinedTlsIdentifier, &sync.Once{})
```

Comment:
Is there any reason to ignore this error?

Reply:
the second argument according to the doc is just a flag that indicates load result status, details https://golang.org/src/sync/map.go?s=6882:6965#L189. In our case we only interested in the value for the given key.